samples/TagHelperSample.Web/Views/Movies/Index.cshtml

```diff
@@ -35,7 +35,7 @@
     </div>
     <div class="sidebar">
         <cache expires-after="TimeSpan.FromMinutes(20)">
-            @await Component.InvokeAsync("FeaturedMovies")
+            @Component.Invoke("FeaturedMovies")
```

Comment:
I changed it as the ViewComponent does not have a `InvokeAsync` method as such(yeah I know the view component has a fallback logic)..